### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by Paulinelali**
This pull request is to add a new section which is the contact form section

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,36 +7,241 @@
...
+    <footer>
+        <section class="footer-section pad-aside">
+            <div class="footer-text">
+                <h1>Contact me</h1>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                    I'd love to help with it!
+                </p>
+            </div>
+            <form action="">
```
Comment (LionRouge1) [inline public/index.html]: - [ ] Nice work implementing the contact form, your design looks identic to the Figma design :wink: However to be able to get our user's messages, we need to set the `method` attribute value to be similar to ` https://formspree.io/f/{form_id}` and `action` attribute value to be `post`. The  `form_id` will be provided to you when creating an account with [formspree service](https://formspree.io/html)

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job, you implemented almost all the requirements:+1: 
- You design a very nice website and followed the Figma Design :1st_place_medal: 

## Required Changes ♻️
- [ ] Your license link in your README file leads to a 404 page, Kindly add a license file to your project to make it looks more professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: public/index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply public/index.html]: Fixed.

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @Paulinelali 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**